resources/sass/components/_form.scss

```diff
@@ -0,0 +1,18 @@
+// form
+// -----------------------------------------------------------------------------
+
+// Custom css for forms.
+/**
+ * Required field
+ */
+.form-required:after {
+  content: '';
+  vertical-align: super;
+  display: inline-block;
+  background-image: url(images/required.svg);
```

Comment:
this doesn't seem to work
![image](https://user-images.githubusercontent.com/11507647/156794475-29db40ee-2e47-49b5-b9d5-72e1caef41c8.png)